### ISSUE TITLE
[BUG] Parameters Pragma Is Not Correctly Rendered When There Is No Newline

### ISSUE THREAD

**Issue by kookma** [Jermolene/TiddlyWiki5#7456]
### Describe the bug

Wrong rendering of `\parameters` when  there is no newline at the end definition.

### To Reproduce

1. open https://tiddlywiki.com/prerelease
2. create a tiddler
3. in the first line type `\parameters (a:2, b:3)`
4. Show preview, you see `\parameters (a:2, b:3)`
5. Save tiddler you see rendered pragmas as plain text e.g. `\parameters (a:2, b:3)`
6. Edit tiddler again, press enter (create a new line and leave `\parameters (a:2, b:3)` in its own line)
7. The issue goes away

In summary: I expect to have the same behavior like other pragma e.g like `\define`

### Screenshots

![image](https://github.com/Jermolene/TiddlyWiki5/assets/830394/aaff8920-ec53-4fc3-8e36-681c18a8da19)

`\Parameters`  when no <kbd>enter</kbd> is pressed

### TiddlyWiki Configuration

- Version 5.3.0

### Desktop (please complete the following information):

- OS: Windows 10
- Chrome/Edge 112

**Comment by Jermolene** [comment]
Thank you @kookma there is indeed an inconsistency here; other pragmas do not require a terminating line break to be recognised.
